pkg/audit/service.go

```diff
@@ -39,12 +49,11 @@ func (service *Service) Run(ctx context.Context) (err error) {
 	if err != nil {
 		return err
 	}
-	for _, fail := range failedNodes {
-		service.statdb.RecordFailedAudit(fail)
+	setNodes := service.Reporter.setAuditFailStatus(ctx, failedNodes)
```

Comment:
What about nodes that pass the audit? They still need to be updated.

Reply:
😪you're right